pkg/platform/common/managedk8s/mk8s-cluster.go

```diff
@@ -200,3 +211,39 @@ func (m *ManagedK8sPlatform) ensureKubeconfig(ctx context.Context, clusterInst *
 	}
 	return names, nil
 }
+
+func (m *ManagedK8sPlatform) GetCloudletManagedClusters(ctx context.Context) ([]*edgeproto.CloudletManagedCluster, error) {
+	cmcClusters, err := m.Provider.GetAllClusters(ctx)
+	if err != nil {
+		return nil, err
+	}
+	getInfraClusterName := func(ci *edgeproto.ClusterInst) string {
+		return m.Provider.NameSanitize(k8smgmt.GetCloudletClusterName(ci))
+	}
+	return infracommon.FilterCloudletManagedClusters(m.CommonPf.PlatformConfig.CloudletKey, cmcClusters, m.caches.ClusterInstCache, getInfraClusterName)
+}
+
+func (m *ManagedK8sPlatform) GetCloudletManagedClusterInfo(ctx context.Context, in *edgeproto.ClusterInst) (*edgeproto.CloudletManagedClusterInfo, error) {
```

Comment:
Do we need to spell out the permissions on the managed cluster? It looks like we should at least be able to get nodes, create namespaces, etc.

Reply:
I think the short answer is we need admin access to the cluster.